docs/migrating_versions/index.rst

```diff
@@ -0,0 +1,12 @@
+.. _version_migration_guide:
```

Comment:
I think that this should be a subsection under API reference as we discussed.

Reply:
Changing that to "User guide" to simplify navigation.

Reply:
Done